lib/shoryuken/cli.rb

```diff
@@ -119,6 +119,10 @@ def parse_cli_args(argv)
           opts[:concurrency] = Integer(arg)
         end
 
+        o.on '-i', '--dispatch-interval DECIMAL', 'Dispatch interval' do |arg|
```

Comment:
Is `dispatch-interval` the right name for this parameter?  There are a few calls to dispatch in the codebase, only one of which is governed by this param.

It'd be nice to find a name which more specifically calls out that it sets the interval used when all processor threads are busy.  E.g. `--busy-wait-time` or something?